File: src/IsbInterop.CSharp/DataTypes/Enumerable/TDataSetState.cs
namespace IsbInterop.DataTypes.Enumerable
{
  /// <summary>
  /// Состояние детального раздела.
  /// </summary>
  public enum TDataSetState
  {
    /// <summary>
    /// Режим редактирования.
    /// </summary>
    dssEdit = 0,

    /// <summary>
    /// Режим вставки.
    /// </summary>
    dssInsert = 1,

    /// <summary>
    /// Режим просмотра.
    /// </summary>
    dssBrowse = 2,

    /// <summary>
    /// Режим неактивного набора данных.
    /// </summary>
    dssInActive = 3,
  }
}

File: src/IsbInterop/DataTypes/Enumerable/TDataSetState.vb
Namespace DataTypes.Enumerable
    ''' <summary>
    ''' Состояние детального раздела.
    ''' </summary>
    Public Enum TDataSetState
        ''' <summary>
        ''' Режим редактирования.
        ''' </summary>
        dssEdit = 0

        ''' <summary>
        ''' Режим вставки.
        ''' </summary>
        dssInsert = 1

        ''' <summary>
        ''' Режим просмотра.
        ''' </summary>
        dssBrowse = 2

        ''' <summary>
        ''' Режим неактивного набора данных.
        ''' </summary>
        dssInActive = 3
    End Enum
End Namespace